packages/react/src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -65,7 +65,8 @@ export type SelectPanelProps = {
 
   // TODO: move these to SelectPanel.Overlay or overlayProps
   width?: OverlayProps['width']
-  height?: OverlayProps['height'] | 'fit-content'
+  height?: 'fit-content' // not used, keeping it around temporary for backward compatibility
```

Comment:
Just out of curiosity, what was the reason of wanting to keep this value? 
Since SelectPanel is an experimental component, I would think we can remove this value to make the code simpler and update the existing use case if there is any.